closed/autoconf/custom-hook.m4

```diff
@@ -75,6 +75,11 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
   # at this point with_cmake should either be no, or the name of the cmake command
   if test "x$with_cmake" = xno ; then
     OPENJ9_ENABLE_CMAKE=false
+
+    # Currently, mixedrefs mode is only available with CMake enabled
+    if test "x$OMR_MIXED_REFERENCES_MODE" != xoff ; then
+      AC_MSG_ERROR([--with-mixedrefs=[static|dynamic] requires --with-cmake])
```

Comment:
An extra set of square brackets are needed to retain the ones in `[static|dynamic]`:
```
  AC_MSG_ERROR([[--with-mixedrefs=[static|dynamic] requires --with-cmake]])
```